meta-mender-raspberrypi-demo/README.md

```diff
@@ -0,0 +1,30 @@
+# meta-mender-raspberrypi-demo
+This README file contains information on the contents of the /meta-mender/meta-raspberrypi-demo layer.
+
+Please see the corresponding sections below for details.
+
+## Dependencies
+============
```

Comment:
=====... is not really markdown syntax and produces weird rending in the web-view.